internal/testplanet/satellite.go

```diff
@@ -26,7 +26,7 @@ import (
 	"storj.io/storj/satellite/console"
 	"storj.io/storj/satellite/console/consoleweb"
 	"storj.io/storj/satellite/mailservice"
-	"storj.io/storj/satellite/marketing/marketingweb"
```

Comment:
curious, why the change ?

Reply:
Yea, from the conversation we had last week about the package structure, the service layer is not needed since both `console` and `marketingweb` server needs access to the `offers` table. It feels weird to put `offers` under marketing only.